rkt/rkt.go

```diff
@@ -105,6 +111,11 @@ func init() {
 	cmdRkt.PersistentFlags().BoolVar(&globalFlags.TrustKeysFromHttps, "trust-keys-from-https",
 		true, "automatically trust gpg keys fetched from https")
 
+	// Run this before the execution of each subcommand to set up output
+	cmdRkt.PersistentPreRun = func(cmd *cobra.Command, args []string) {
+		errOut = common.NewErrorSanitizer(os.Stderr, cmd.Name(), globalFlags.Debug)
```

Comment:
`common.NewErrorSanitizer` is only called in the `rkt` binary but not in stage1's binaries (`init` or `gc`). How does it print the errors when the errors are not generated from the `rkt` binary?